Devscord.DiscordFramework/ReactionContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Discord;
+using Discord.WebSocket;
+
+namespace Devscord.DiscordFramework
+{
+    public class ReactionContext
+    {
+        public ISocketMessageChannel Channel { get; }
+        public IEmote Emote { get; }
+        public Optional<SocketUserMessage> Message { get; }
+        public ulong MessageId { get; }
+        public Optional<IUser> User { get; }
```

Comment:
co to jest IUser

Reply:
https://discord.foxbot.me/docs/api/Discord.IUser.html?q=Iuser